tests/Makefile

```diff
@@ -0,0 +1,29 @@
+all: lint unit_tests
+
+lint:
```

Comment:
@antoviaque Several problems here.

1. If we use `wget` to download the file from the instance manager each time the tests are run then we can break the tests in this repo by making a change to the other repo.  I don't think we want this, so instead I suggest to add a make target that updates the files from the instance manager.  We will have to do this manually, but nothing bad happens if we don't do it, and it prevents our tests from randomly breaking.

2. The `pylintrc` from the instance manager can't be used unmodified, since it includes `load-plugins=lint_django`, which won't work for this repo.  I tried overriding this from the command line, but the plugins given on the command line are loaded in addition to those in the configuration file, so there is nothgin I can do.  This is another reason why we can't download the file from the instance manager.

3. I can't make prospector work when using the same configuration as the instance manager.  I runs fine and claims that all is good, but if I modify the files to contain pylint violations, they don't show up in the output.  If I run pylint manually, all works well, and when I add violations of e.g. pyflakes I can see them as well.  Prospector doesn't offer any debug output, so it's very hard to tell what's going on.  It _claims_ to use the correct pylint configuration file, but I can't see any indication that pylint is actually being run.  The next step here would be to dig into the prospector code to figure out what it's actually doing.

How much additional time am I supposed to invest in this?  I personally don't think the change to use exactly the same configuration adds much value over the current version, which is already mostly consistent with the instance manager, and we only have like 200 lines of code in this repo.